### PR TITLE
Add info for Factory access and scope

### PR DESCRIPTION
**Pull request by kprosise**
Information on scope, especially for devices, as well the ways that account roles effect scope.

QA steps taken was linkcheck and checking documentation rendering locally.
Also used vale linter and spellcheck.

This commit addresses Jira FS #1080

Signed-off-by: Katrina Prosise <katrina.prosise@foundries.io>

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

Added some additional information as requested by customers.
It was initially requested for "roles" however some information was a better fit under teams,
which deals with scope

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

Some details were filled in from implicit/between the lines info, so it may be good to fact check these documents.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 9ed3a5e76020034395fdc60e51a62a19a76d0030 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1775/docs/artifacts/html/index.html

File: source/user-guide/account-management/team-based-access.rst [hunk 1]
```diff
@@ -3,40 +3,54 @@
...
+Larger organizations often need to restrict the access level a user has to the Factory.
+For example, some users might only need access for managing devices,
+while other users only need read-only access to source code.
+Factory **Teams** is how FoundriesFactory enables you to control access.
 
- * **Access control** - What :ref:`scopes <ref-scopes>` are granted to
-   the team. e.g. ``source:read``.
+A Team is comprised of:
+
+ * **Access control** - What :ref:`scope <ref-scopes>` is granted to
```
Comment (doanac) [inline source/user-guide/account-management/team-based-access.rst:16]: i believe "scope" should be plural here. A team normal has multiple scopes

File: source/user-guide/account-management/team-based-access.rst [hunk 2]
```diff
@@ -3,40 +3,54 @@
...
+   the team, e.g. ``source:read``.
 
  * **Members** - What users belong to the team.
 
-Once a user is assigned a team, their access will be limited to the
-scopes granted to that team.
+Once a user is assigned a team, their access will be limited to the scope granted to that team.
```
Comment (doanac) [inline source/user-guide/account-management/team-based-access.rst:21]: scope->scopes

File: source/user-guide/account-management/team-based-access.rst [hunk 3]
```diff
@@ -3,40 +3,54 @@
...
+   the team, e.g. ``source:read``.
 
  * **Members** - What users belong to the team.
 
-Once a user is assigned a team, their access will be limited to the
-scopes granted to that team.
+Once a user is assigned a team, their access will be limited to the scope granted to that team. 
+
+.. important::
+   It may take a moment for changes to scope/teams to take effect.
```
Comment (doanac) [inline source/user-guide/account-management/team-based-access.rst:24]: Let's check with @MiloCasagrande on this for the webui. You may have to log out/in in the web UI. However, I don't think API scopes require any time lapse.

File: source/user-guide/account-management/team-based-access.rst (hunk 1 above)
Comment (kprosise) [inline source/user-guide/account-management/team-based-access.rst:16]:
```suggestion
 * **Access control** - What :ref:`scopes <ref-scopes>` are granted to
```

File: source/user-guide/account-management/team-based-access.rst (hunk 2 above)
Comment (kprosise) [inline source/user-guide/account-management/team-based-access.rst:21]:
```suggestion
Once a user is assigned a team, their access will be limited to the scopes granted to that team. 
```

**Review by kprosise — COMMENTED** [review]
change scope -> scopes in two instances

**Comment by doanac** [comment]
Docs for 4e3ce02bd09242d59c492aedd5f5f0588c75b4c3 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1776/docs/artifacts/html/index.html

File: source/user-guide/account-management/team-based-access.rst (hunk 3 above)
Comment (MiloCasagrande) [reply source/user-guide/account-management/team-based-access.rst:24]: Logging out/in unfortunately will not delete the member access cache: the cache key is built in a way that's not possible to retrieve it via the user id. That can be changed though.

Comment (kprosise) [reply source/user-guide/account-management/team-based-access.rst:24]: @MiloCasagrande Should I keep in the bit about it taking a moment then?

Comment (MiloCasagrande) [reply source/user-guide/account-management/team-based-access.rst:24]: I think it's ok to say it will take some minutes for the changes to apply.

Comment (kprosise) [reply source/user-guide/account-management/team-based-access.rst:24]: Okay, I'm going to mark this conversation as resolved. Thanks @MiloCasagrande and @doanac

**Review by vanmaegima — APPROVED** [review]
LGTM

**Pull request merged**